.golangci.yml

```diff
@@ -0,0 +1,67 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+  golint:
+    min-confidence: 0
+  maligned:
+    suggest-new: true
+  misspell:
+    locale: UK
+  gocritic:
+    enabled-tags:
+      - performance
+      - style
+      - experimental
+    disabled-checks:
+      - sloppyReassign
+      - docStub
+      - paramTypeCombine
+
+linters:
+  enable-all: true
+  disable:
+    - dupl
+    - gochecknoglobals
+    - gocyclo
+    - goconst
+    - lll
+    - misspell
```

Comment:
that's a good one to have imo.

In fact I'd keep:
lll, misspell, gochecknoglobals, and gocyclo.
You can update gocyclo to higher values in gocyclo specific settings;
you can also "exclude" individual lines / files with //nolint:lll for instance

Reply:
That's a great point, 
I've updated to turn on `misspell`, `gocyclo` and `goconst`

It looks like it'd be quite a bit more work to enable `lll` and `gochecknoglobals` so I'll raise those as issues